org.osgi.util.converter/src/org/osgi/util/converter/package-info.java

```diff
@@ -31,10 +31,10 @@
  * Example import for providers implementing the API in this package:
  * <p>
  * {@code  Import-Package: org.osgi.util.converter; version="[1.0,1.1)"}
- * 
+ *
  * @author $Id$
  */
-@Version("1.0.2")
```

Comment:
is this just by accident to go from `@Version("1.0.2")` to `@Version("1.0.8")`?

Reply:
The version number in Felix has been updated more often than the one in OSGi with a micro version (meaning a patch, non-functional update).
In the end it doesn't really matter if we decide that we have to go to 1.1.x anyway because of the discussion here https://github.com/osgi/osgi/issues/227#issuecomment-849719644

Reply:
For me, 1.0.8 is fine since it reflects the version from Apache.